app/models/registerable_edition.rb

```diff
@@ -14,7 +14,7 @@ def slug
     # to be consistent with Panopticon's slug format.
     panopticon_slug = Whitehall.url_maker.public_document_path(edition).sub(/\A\//, "")
     if edition.deleted?
-      panopticon_slug.sub!(%r{/deleted-([^/]+)$}, '/\1')
+      panopticon_slug.sub!(%r{deleted-([^/]+)$}, '\1')
```

Comment:
Possibly for safety, yes. But it's also `sub` instead of `gsub`, and it's only for deleted editions. But yes, worth anchoring it anyway.